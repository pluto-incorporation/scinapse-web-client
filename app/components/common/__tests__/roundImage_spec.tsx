jest.unmock("../roundImage");

import * as React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import RoundImage, { IRoundImageProps } from "../roundImage";

describe("<RoundImage /> Component", () => {
  let roundImageWrapper: ShallowWrapper<IRoundImageProps>;
  const mockWidth = 50;
  const mockHeight = 50;

  beforeEach(() => {
    roundImageWrapper = shallow(<RoundImage width={mockWidth} height={mockHeight} />);
  });

  it("should match snapshot", () => {
    expect(roundImageWrapper.html()).toMatchSnapshot();
  });
});
