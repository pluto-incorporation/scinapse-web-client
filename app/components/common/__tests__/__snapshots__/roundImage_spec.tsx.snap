// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoundImage /> Component should match snapshot 1`] = `"<div class=\\"imgWrapper\\"><img style=\\"width:50px;height:50px\\" class=\\"roundImage\\"/></div>"`;
